apps/admin/src/pages/Users/index.tsx

```diff
@@ -1,10 +1,184 @@
+              height={'100%'}
+              style={{
+                borderRadius: '50%',
+              }}
+            />
+          </div>
+
+          <div className='flex flex-col w-full'>
+            <span className='w-full text-gray-700 font-bold text-[17px]'>
+              {info?.getValue()}
+            </span>
+            <div className='gap-[2px] flex flex-col'>
+              <span className='w-full text-gray-700 font-medium text-[15px]'>
+                {info?.row?.original?.email}
+              </span>
+
+              <Link
+                href={`https://github.com/` + info?.row?.original?.github}
+                passHref
+              >
+                <a target='_blank' rel='noopener noreferrer'>
+                  <span className='w-full flex flex-row items-center gap-[5px]'>
+                    <i className='w-[20px] h-[20px]'>
+                      <GithubIcon className='fill-[#a3aed0]' />
+                    </i>
+                    <span className='text-gray-700 font-bold text-[14px]'>
+                      {info?.row?.original?.github}
+                    </span>
+                  </span>
+                </a>
+              </Link>
+            </div>
+          </div>
+        </div>
+      );
+    },
+  },
+  {
+    accessorKey: 'projects',
+    header: () => <span>Projects</span>,
+    cell: (info) => {
+      return (
+        <div className=''>
+          <span className='text-gray-700 font-bold text-[14px]'>
+            {info?.row?.original?.projects?.length}
+          </span>
+        </div>
+      );
+    },
+  },
+  {
+    accessorKey: 'createdAt',
+    header: () => <span>Created Date</span>,
+    cell: (info) => {
+      return (
+        <span className='text-gray-700 font-bold text-[14px]'>
+          {new Date(info?.getValue()).toDateString()}
+        </span>
+      );
+    },
+  },
+  {
+    accessorKey: 'role',
+    header: () => <span>Role</span>,
+    cell: (info) => {
+      const selectedValue = {
+        value: info.getValue(),
+        label: info.getValue(),
+        isDisabled: true,
+      };
+
+      const options = [
+        {
+          value: 'USER',
+          label: 'USER',
+          isDisabled: info.getValue() == 'USER' ? true : false,
+        },
+        {
+          value: 'ADMIN',
+          label: 'ADMIN',
+          isDisabled: info.getValue() == 'ADMIN' ? true : false,
+        },
+      ];
+      return (
+        <span className='text-gray-700 font-bold text-[14px]'>
+          <Select
+            isSearchable={false}
+            options={options}
+            value={selectedValue}
+            styles={styles}
+          />
+        </span>
+      );
+    },
+  },
+  {
+    accessorKey: 'action',
+    header: () => <span>Action</span>,
+    cell: (info) => {
+      return (
+        <button
+          className={classNames(
+            'text-[14px] text-white font-bold py-[5px] px-[20px] rounded-full',
+            {
+              'bg-red-600': true,
```

Comment:
same as above, just pass it on the string

Reply:
done